web/src/settings_components.ts

```diff
@@ -499,7 +499,6 @@ const dropdown_widget_map = new Map<string, DropdownWidget | null>([
     ["realm_create_multiuse_invite_group", null],
     ["can_remove_subscribers_group", null],
     ["realm_can_access_all_users_group", null],
-    ["can_mention_group", null],
```

Comment:
`new_group_can_mention_group_widget` variable and `set_new_group_can_mention_group_widget` function can also be removed now I guess.

Reply:
[Fixed](https://github.com/zulip/zulip/compare/4cbe6a2e6c244026bd3c7d5d83043a812e3e37ba..5018be3a0770d60ddf81cec3863722fab4e7ab5c)